src/components/button/button.style.ts

```diff
@@ -67,6 +71,30 @@ function stylingForType({
       font-size: 16px;
       min-height: 48px;
     `}
+
+    /* BUTTON MINOR */
+    ${isMinor &&
+    size === "small" &&
+    css`
+      min-height: 32px;
+      padding: var(--spacing000) var(--spacing100) var(--spacing000)
```

Comment:
These values will present consumers being able to set their own custom padding values as it is applied after the `space` [styling](https://github.com/Sage/carbon/blob/b9ff6740d3e1987d6361c3f2a446d28c1afa9fad/src/components/button/button.style.ts#L108). They will need to be the other way around imo so they act as defaults that can be overridden